mobile-widgets/qmlmanager.cpp

```diff
@@ -352,6 +362,10 @@ void QMLManager::copyAppLogToClipboard()
 		QTextStream in(&f);
 		copyString += in.readAll();
 	}
+#ifdef ENABLE_STARTUP_TIMING
+	copyString += "---------- startup timer ----------\n";
+	copyString += stpText;
+#endif
 	copyString += "---------- finish ----------\n";
```

Comment:
we should just write the logging output among the rest of the logging output.

Reply:
And how do I write logging output before the log file is opened (subsurface-mobile-main.cpp) ?